src/messages.css

```diff
@@ -0,0 +1,41 @@
+body {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+}
+
+form {
```

Comment:
Using element name selectors is not desired because in a document, there could be multiple forms. If you target them all, you could break the style for specific ones. So here we should be targetting the form by either id or class. If you aren't sure, always default to using classes. That way, only this specific form style is changed; all the other form elements remain unchanged / unaffected by this change.

```
#message-form {

or

.message-form {
```

Then in HTML:

```
<form id="message-form"

OR

<form class="message-form"
```

Reply:
Ok I will update everything

Reply:
I've updated eveything as per request